test.py

```diff
@@ -1,6 +1,6 @@
 if __name__ == "__main__":
     import pytest
-    from homematicip import __version__
+    from homematicip import _version
```

Comment:
Can you please revert the changes from test.py and test_aio.py. They are not used anymore and should be deleted.